src/init.cpp

```diff
@@ -1235,23 +1235,18 @@ bool AppInit2(Config &config, thread_group &threadGroup)
     fFeeEstimatesInitialized = true;
 
     // Set EB and MAX_OPS_PER_SCRIPT for the SV chain
-    if (IsSv2018Scheduled())
+    if (AreWeOnSVChain() && IsSv2018Activated(Params().GetConsensus(), chainActive.Tip()))
     {
-        if (IsSv2018Enabled(Params().GetConsensus(), chainActive.Tip()))
-        {
-            maxScriptOps = SV_MAX_OPS_PER_SCRIPT;
-            excessiveBlockSize = SV_EXCESSIVE_BLOCK_SIZE;
-            settingsToUserAgentString();
-        }
+        maxScriptOps = SV_MAX_OPS_PER_SCRIPT;
+        excessiveBlockSize = SV_EXCESSIVE_BLOCK_SIZE;
+        settingsToUserAgentString();
```

Comment:
shouldn't you set canonicaltxorder here to false here? I though I saw earlier you set the global tweak to true?

Reply:
you're right will fix